angular.json

```diff
@@ -7,7 +7,8 @@
       "projectType": "application",
       "schematics": {
         "@schematics/angular:component": {
-          "style": "scss"
+          "style": "scss",
+          "skipTests": true
```

Comment:
Good job adding the skipTests rule, so that we can avoid deleting the tests files from now on, as we are not using tests